fiat-roles/src/main/java/com/netflix/spinnaker/fiat/config/RedisConfig.java

```diff
@@ -48,6 +40,11 @@ public JedisPool jedisPool(@Value("${redis.connection:redis://localhost:6379}")
     return createPool(redisPoolConfig, connection, timeout);
   }
 
+  @Bean
+  RedisClientDelegate redisClientDelegate(JedisPool jedisPool) {
+    return new JedisClientDelegate(jedisPool);
+  }
```

Comment:
Not required, but raising your attention if you missed it: If you want all of the kork auto-configuration stuff (interchangeable redis/dynomite configs, named client selector, etc), you can also do: https://github.com/spinnaker/kork/blob/master/kork-jedis/src/main/java/com/netflix/spinnaker/kork/jedis/JedisClientConfiguration.java